cypress/integration/smoke/SmokeLandingPage.ts

```diff
@@ -0,0 +1,43 @@
+// 
+// Filename: SmokeLandingPage.ts
+// Execution: yarn cypress run -s /Users/filip/Desktop/DXDAO/DAVI-QA/cypress/integration/smoke/SmokeLandingPage.ts
+// 
+
+import { LandingPage } from "../../support/pageObjects/LandingPage";
+import configDevelopmentJSON from '../../config/development.json';
+
+describe('Check Landing Page', () => {
+  beforeEach(() => {
+    cy.visit(configDevelopmentJSON.baseUrl)
```

Comment:
I think this is not the best way to pull config variables. Since config file should be set when running `cypress run`, config/env variables are available from `Cypress.config('baseUrl')` here